src/osx_clipboard.rs

```diff
@@ -61,6 +61,55 @@ impl ClipboardProvider for OSXClipboardContext {
             Ok(string_array[0].as_str().to_owned())
         }
     }
+    fn get_binary_contents(&mut self) -> Result<ClipboardContent, Box<Error>> {
+        let string_class: Id<NSObject> = {
+            let cls: Id<Class> = unsafe { Id::from_ptr(class("NSString")) };
+            unsafe { transmute(cls) }
+        };
+        let image_class: Id<NSObject> = {
+            let cls: Id<Class> = unsafe { Id::from_ptr(class("NSImage")) };
+            unsafe { transmute(cls) }
+        };
+        let url_class: Id<NSObject> = {
+            let cls: Id<Class> = unsafe { Id::from_ptr(class("NSURL")) };
+            unsafe { transmute(cls) }
+        };
+        let classes = vec![url_class, image_class, string_class];
```

Comment:
Note that the more "specific" class needs to come first, as `readObjectsForClasses` will try to match in order